test/node/source.test.coffee

```diff
@@ -34,12 +34,12 @@ describe "Node Source", ->
 
     it "has an array of all versions", ->
       assert.equal typeof(this.s.all), "object"
-      assert.equal this.s.all.length, 215
-      assert.equal this.s.all[214], '0.11.13'
+      assert.equal this.s.all.length, 219
```

Comment:
3 new versions added (`4.0.0`, `4.1.0` & `4.2.0-alpha.3`) + `0.10.16-isaacs-manual` is now matched as well.